src/main/java/uk/gov/pay/adminusers/app/config/AdminUsersModule.java

```diff
@@ -77,6 +77,7 @@ private NotificationService provideUserNotificationService() {
                 new NotifyClientProvider(configuration.getNotifyConfiguration(), getSSLContext()),
                 configuration.getNotifyConfiguration().getSecondFactorSmsTemplateId(),
                 configuration.getNotifyConfiguration().getInviteEmailTemplateId(),
+                configuration.getNotifyConfiguration().getForgottenPasswordEmailTemplateId(),
```

Comment:
Addressed in https://github.com/alphagov/pay-adminusers/pull/74/commits/ffddd2eb0c6381b040f23dd4d6a422a1be849a3a